contributors/design-proposals/local-storage-overview.md

```diff
@@ -196,37 +198,46 @@ Since local PVs are only accessible from specific nodes, the scheduler needs to
    volumes:
    - name: myEmptyDir
      emptyDir:
-		sizeLimit: 2Gi
+       sizeLimit: 2Gi
   ```
 
-6. It is recommended to require `limits` to be specified for `storage` in all pods. `storage` will not affect the `QoS` Class of a pod since no SLA is intended to be provided for storage capacity isolation. it is recommended to use Persistent Volumes as much as possible and avoid primary partitions.
+6. It is recommended to require `limits` to be specified for `storage` in all pods. `storage` will not affect the `QoS` Class of a pod since no SLA is intended to be provided for storage capacity isolation. It is recommended to use Persistent Volumes as much as possible and avoid primary partitions.
 
 ### Alice manages a Database which needs access to “durable” and fast scratch space
 
 1. Cluster administrator provisions machines with local SSDs and brings up the cluster
-2. When a new node instance starts up, an addon DaemonSet discovers local “secondary” partitions which are mounted at a well known location and creates Local PVs for them if one doesn’t exist already. The PVs will include a path to the secondary device mount points, and a hostname label ties the volume to a specific node.  A StorageClass is required and will have a new optional field `toplogyKey`.  This field tells the scheduler to filter PVs with the same `topologyKey` value on the node. The `topologyKey` can be any label key applied to a node.  For the local storage case, the `topologyKey` is `kubernetes.io/hostname`, but the same mechanism could be used for zone constraints as well.
+2. When a new node instance starts up, an addon DaemonSet discovers local “secondary” partitions which are mounted at a well known location and creates Local PVs for them if one doesn’t exist already. The PVs will include a path to the secondary device mount points, and a node affinity annotation ties the volume to a specific node.  The annotation tells the scheduler to filter PVs with the same affinity key/value on the node.  For the local storage case, the key is `kubernetes.io/hostname`, but the same mechanism could be used for zone constraints as well.
 
     ```yaml
     kind: StorageClass
     apiVersion: storage.k8s.io/v1
     metadata:
       name: local-fast
-    provisioner: ""
     toplogyKey: kubernetes.io/hostname
     ```
     ```yaml
     kind: PersistentVolume
     apiVersion: v1
     metadata:
       name: local-pv-1
-      labels:
-        kubernetes.io/hostname: node-1
+      annotations:
```

Comment:
Let's have this doc reflect the eventual state of the API that we want.  So let's make this a field instead of an annotation.